libraries/messageHandler.py

```diff
@@ -99,12 +111,16 @@ async def handleMessage(message, client, commandMessageStarter):
                     await commands.displayBoard(message, renderedBoard)
                 elif action == 'players':
                     await commands.showPlayerStatistics(message, jsonManager.readJson(), client)
+                elif action == 'dm':

```

Comment:
wait is this copy-paste from line 66

Reply:
Yes, it is the same as line 66 as one is a command for an already started game and one is a command from the lobby, but both do the same action as of now